packages/flutter/test_private/pubspec.yaml

```diff
@@ -0,0 +1,24 @@
+name: flutter_test_private
+author: Flutter Authors <flutter-dev@googlegroups.com>
+description: Tests private interfaces of the flutter
+
+environment:
+  # The pub client defaults to an <2.0.0 sdk constraint which we need to explicitly overwrite.
+  sdk: ">=2.10.0-0.0.dev <3.0.0"
+
+dependencies:
+  # To update these, use "flutter update-packages --force-upgrade".
+  args: 1.6.0
```

Comment:
you don't actually use args, it seems

Reply:
Right. Removed. (I would if there was a nullsafety version.)